internal/grpc/grpc.go

```diff
@@ -135,17 +136,50 @@ func (s *Service) GetRequestStruct(id, environmentID string) (string, error) {
 		return "", err
 	}
 
-	request := dynamicpb.NewMessage(md.Input())
-	reqJSON, err := (protojson.MarshalOptions{
-		Indent:          "  ",
-		EmitUnpopulated: true,
-		UseProtoNames:   true,
-	}).Marshal(request)
+	jsonBytes, err := json.MarshalIndent(generateExampleJSON(md.Input()), "", "  ")
 	if err != nil {
+		fmt.Println("Error marshaling to JSON:", err)
 		return "", err
```

Comment:
I'm surprised by this println

Maybe the error could be simply wrapped


```suggestion
 		return "", fmt.Errorf("Error marshaling to JSON:", err)
```

Or something equivalent to c.showError